quipucords/scanner/vcenter/tests_vc_inspect.py

```diff
@@ -184,15 +184,18 @@ def test_parse_vm_props(self, mock_dt):
 
         ip_addresses, mac_addresses = ['1.2.3.4'], ['00:50:56:9e:09:8c']
 
-        facts = {'name': 'vm1',
-                 'guest.net': '',  # mac/ip addr returned by get_nics
-                 'summary.runtime.powerState': 'poweredOn',
-                 'summary.guest.hostName': 'hostname',
-                 'summary.config.guestFullName': 'Red Hat 7',
-                 'summary.config.memorySizeMB': 1024,
-                 'summary.config.numCpu': 4,
-                 'summary.config.uuid': '1111',
-                 'runtime.host': ''}  # runs through host_facts values
+        facts = {
+            "name": "vm1",
+            "guest.net": "",  # mac/ip addr returned by get_nics
+            "summary.runtime.powerState": "poweredOn",
+            "summary.guest.hostName": "hostname",
+            "summary.config.guestFullName": "Red Hat 7",
+            "summary.config.memorySizeMB": 1024,
+            "summary.config.numCpu": 4,
+            "summary.config.uuid": "1111",
+            "runtime.host": "",
+            "config.template": True,
+        }  # runs through host_facts values
```

Comment:
for reviewing purposes is better to just add new code w/o reformatting old stuff.
don't need to change anything, though.